src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,17 @@
+<footer class="pmd-footer bg-dark pmd-footer-dark">
+    <div class="container-fluid">
+        
+    </div>
+    <mat-toolbar>
+        <div class="text-center">
+            © 2023  <a href="#"><strong>TimeTravellers</strong></a> 
+        </div>
+        <span style="flex: 1 1 auto"></span>
+        
+        
+        <button mat-button (click)="goToPage()">
+            <mat-icon aria-hidden="false" fontIcon="contacts" href="/contact"></mat-icon>
+            Contact-us
+        </button>
+    </mat-toolbar>
+</footer>
```

Comment:
```suggestion
    <mat-toolbar class="footer">
        <div class="text-center">
            © 2023  <a routerLink=""><strong>TimeTravellers</strong></a> 
        </div>
        <span style="flex: 1 1 auto"></span>
        
        
        <button mat-button routerLink="/contact">
            <mat-icon aria-hidden="false" fontIcon="contacts" ></mat-icon>
            Contact-us
        </button>
    </mat-toolbar>
```

Use routerLink, nor href